Gemfile.lock

```diff
@@ -266,125 +268,127 @@ GEM
       zeitwerk (~> 2.3)
-    addressable (2.8.4)
+    addressable (2.8.6)
       public_suffix (>= 2.0.2, < 6.0)
-    airbrake (11.0.1)
+    airbrake (11.0.3)
       airbrake-ruby (~> 5.1)
-    airbrake-ruby (5.2.0)
+    airbrake-ruby (5.2.1)
       rbtree3 (~> 0.5)
     ansible (0.2.2)
     ar_multi_threaded_transactional_tests (0.5.0)
       activerecord (>= 4.2.0, < 6.2.0)
-    ast (2.4.1)
+    ast (2.4.2)
     attr_encrypted (3.1.0)
       encryptor (~> 3.0.0)
-    audited (5.4.2)
+    audited (5.4.3)
       activerecord (>= 5.0, < 7.2)
       request_store (~> 1.2)
-    autoprefixer-rails (9.4.8)
+    autoprefixer-rails (9.8.6.5)
       execjs
-    awesome_print (1.6.1)
-    aws-eventstream (1.2.0)
-    aws-partitions (1.329.0)
-    aws-sdk-core (3.100.0)
-      aws-eventstream (~> 1, >= 1.0.2)
-      aws-partitions (~> 1, >= 1.239.0)
+    awesome_print (1.9.2)
+    aws-eventstream (1.3.0)
+    aws-partitions (1.887.0)
+    aws-sdk-core (3.191.0)
+      aws-eventstream (~> 1, >= 1.3.0)
+      aws-partitions (~> 1, >= 1.651.0)
+      aws-sigv4 (~> 1.8)
+      jmespath (~> 1, >= 1.6.1)
+    aws-sdk-ecr (1.69.0)
+      aws-sdk-core (~> 3, >= 3.191.0)
       aws-sigv4 (~> 1.1)
-      jmespath (~> 1.0)
-    aws-sdk-ecr (1.3.0)
-      aws-sdk-core (~> 3)
-      aws-sigv4 (~> 1.0)
-    aws-sdk-kms (1.23.0)
-      aws-sdk-core (~> 3, >= 3.58.0)
+    aws-sdk-kms (1.77.0)
+      aws-sdk-core (~> 3, >= 3.191.0)
       aws-sigv4 (~> 1.1)
-    aws-sdk-s3 (1.45.0)
-      aws-sdk-core (~> 3, >= 3.58.0)
+    aws-sdk-s3 (1.143.0)
+      aws-sdk-core (~> 3, >= 3.191.0)
       aws-sdk-kms (~> 1)
-      aws-sigv4 (~> 1.1)
-    aws-sigv4 (1.6.1)
+      aws-sigv4 (~> 1.8)
+    aws-sigv4 (1.8.0)
       aws-eventstream (~> 1, >= 1.0.2)
-    base64 (0.1.1)
+    base64 (0.2.0)
+    bigdecimal (3.1.6)
     binding_of_caller (0.8.0)
       debug_inspector (>= 0.0.1)
-    bootsnap (1.4.5)
-      msgpack (~> 1.0)
+    bootsnap (1.18.3)
+      msgpack (~> 1.2)
     bootstrap-sass (3.4.1)
       autoprefixer-rails (>= 5.2.1)
       sassc (>= 2.0.0)
     bootstrap3-datetimepicker-rails (4.17.47)
       momentjs-rails (>= 2.8.1)
     brakeman (5.4.1)
     builder (3.2.4)
-    bundler-audit (0.9.0.1)
+    bundler-audit (0.9.1)
       bundler (>= 1.2.0, < 3)
       thor (~> 1.0)
     byebug (8.2.5)
-    chef-utils (18.1.29)
+    chef-utils (18.4.2)
       concurrent-ruby
-    coderay (1.1.1)
+    coderay (1.1.3)
     commonmarker (0.23.10)
-    concurrent-ruby (1.2.2)
-    connection_pool (2.2.1)
-    crack (0.4.3)
-      safe_yaml (~> 1.0.0)
+    concurrent-ruby (1.2.3)
+    connection_pool (2.4.1)
+    crack (0.4.6)
+      bigdecimal
+      rexml
     crass (1.0.6)
-    dalli (3.2.3)
+    dalli (3.2.7)
+      base64
     ddtrace (0.45.0)
       msgpack
     debug_inspector (0.0.3)
     diffy (3.4.2)
-    dogstatsd-ruby (3.0.0)
-    domain_name (0.5.20190701)
-      unf (>= 0.0.5, < 1.0.0)
-    doorkeeper (5.6.6)
+    dogstatsd-ruby (3.3.0)
+    domain_name (0.6.20240107)
```

Comment:
### [domain_name](https://github.com/knu/ruby-domain_name)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 0.5.20190701 → 0.6.20240107

[_~~change-log~~, [source-code](https://github.com/knu/ruby-domain_name), [gem-diff](https://my.diffend.io/gems/domain_name/0.5.20190701/0.6.20240107)_]





<details>
<summary>Commits</summary>

A change of **14** commits. See the full changes on [the compare page](https://github.com/knu/ruby-domain_name/compare/v0.5.20190701...v0.6.20240107).

These are the first 10 commits:
- (17ae27b) [Avoid use of `<<~WORD` as it causes syntax error on older ru…](https://github.com/knu/ruby-domain_name/commit/17ae27bbe6c5e21ce97b9137537f97f8e9329e4d)
- (8d1c3e6) [OK, 1.8.7 is still supported](https://github.com/knu/ruby-domain_name/commit/8d1c3e66ce4d904f854ab9ddf42193f8d543c744)
- (c64a590) [Add ruby 2.4-2.6 and retire 2.0-2.3 and rbx-2](https://github.com/knu/ruby-domain_name/commit/c64a59027939aa34e1f5f0efc5cb654d73ccb966)
- (4d08512) [Require Ruby >=2.7 and drop the dependency on unf](https://github.com/knu/ruby-domain_name/commit/4d08512da35ce1edb156ef70ed4781b9664a0f26)
- (88b13c2) [Migrate the CI to GitHub Actions](https://github.com/knu/ruby-domain_name/commit/88b13c2d117bafccf07918338466d3807a0b2370)
- (9701b77) [Bump the minor version to 0.6](https://github.com/knu/ruby-domain_name/commit/9701b77ad2c464c4a772629302fb1f9bccadaac6)
- (c2c0e97) [Update the changelog](https://github.com/knu/ruby-domain_name/commit/c2c0e978e726856e60d1e132f34df6a3452381ef)
- (d64fd23) [Update the eTLD database to 2023-11-09 00:10:10 UTC.](https://github.com/knu/ruby-domain_name/commit/d64fd2307f2796fd180b908666a782857b941525)
- (567c4cd) [allows DomainName.normalize to accept ASCII-only non-unicode…](https://github.com/knu/ruby-domain_name/commit/567c4cd385d2535626ce8d7d190b22e07772564e)
- (e60aae9) [Merge pull request #27 from tisba/fix/issue-23](https://github.com/knu/ruby-domain_name/commit/e60aae96a33471f93d588f6df10d62293721d783)


</details>